msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AadInstanceDiscoveryResponse.java

```diff
@@ -29,4 +29,13 @@ class AadInstanceDiscoveryResponse {
 
     @JsonProperty("correlation_id")
     private String correlationId;
+
+    @JsonProperty("authorization_endpoint")
+    private String authorizationEndpoint;
```

Comment:
I recommend you create a new response class for OIDC discovery instead of reusing the Instance Discovery class.

Reply:
In the latest commit these fields are now in a new `OidcDiscoveryResponse` class.